test/e2e/storage/testsuites/snapshottable.go

```diff
@@ -184,20 +184,26 @@ func (s *snapshottableTestSuite) DefineTests(driver storageframework.TestDriver,
 			nodeName := pod.Spec.NodeName
 			gomega.Expect(nodeName).NotTo(gomega.BeEmpty(), "pod.Spec.NodeName must not be empty")
 
-			ginkgo.By(fmt.Sprintf("[init] waiting until the node=%s is not using the volume=%s", nodeName, pv.Name))
+			// When checking a node's volumeInUse this can be one of two values:
+			// - the pv.Name for in-tree volumes
```

Comment:
add a comment here that intree doesn't support snapshots

Reply:
This has been included.